### ISSUE TITLE
Dual Ivb-E kernel panic

### ISSUE THREAD

**Issue by BrushXue** [acidanthera/bugtracker#1070]
OpenCore 0.59, latest kexts.
CPU: dual E5-2667 v2
Motherboard: Asrock EP2C602-4L/D16
My configuration can work with Clover. Now I'm switching to OpenCore (following the tutorial in https://dortania.github.io/OpenCore-Install-Guide/config.plist/legacy/sandy-bridge-e.html).
I got the following error :
`panic(cpu 0 caller 0x......): "x86_validate topology() 32 threads but 33 registered from MADT~0/...`
If you google that info, the thread was started by myself. Enabling the `patchAPIC` option in Clover can fix it.
I guess the problem is OpenCore couldn't fix my MADT table automatically?

Thanks

**Comment by vit9696** [comment]
Correct, you should drop your own MADT and insert a patched one. Here is the relevant [code in Clover](https://github.com/CloverHackyColor/CloverBootloader/blob/d1fbcb92bfc00d92a7797e5e8a1acec7eed65a47/rEFIt_UEFI/Platform/AcpiPatcher.cpp#L2182-L2220). CC @khronokernel

**Comment by khronokernel** [comment]
Added an issue in the Dortania bugtracker, @BrushXue could you add your ACPI dump there? Would be nice to document the process for others to patch themselves
